CHANGELOG.md

```diff
@@ -12,7 +12,7 @@ Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how
 ## [elrond-wasm 0.17.4] - 2021-06-30
 - conversions from big ints to small int: `BigUint::to_u64`, `BigInt::to_i64`
 
-## [elrond-wasm 0.17.3] - 2021-06-11
+## [elrond-wasm 0.17.4] - 2021-06-11
```

Comment:
Why change a past changelog? Should add another entry for next version 0.17.5

Reply:
Mass replaced, didn't notice I also changed it here. Fixed.